cvat/apps/engine/serializers.py

```diff
@@ -1207,6 +1206,65 @@ class SubLabeledShapeSerializer(ShapeSerializer, AnnotationSerializer):
 class LabeledShapeSerializer(SubLabeledShapeSerializer):
     elements = SubLabeledShapeSerializer(many=True, required=False)
 
+class LabeledImageSerializerFromDB(serializers.BaseSerializer):
+    def to_representation(self, instance):
+        def convert_tag(tag):
+            tag['attributes'] = [OrderedDict(attr) for attr in tag['labeledimageattributeval_set']]
```

Comment:
Why is `OrderedDict` used here?

Reply:
To convert these values from ``dotdict`` type back to OrderedDict
Otherwise tests fail because it calls redefined ``__eq__`` function from ``dotdict`` class which is not supposed to be used there.

Reply:
Makes sense. Would it be possible to use regular `dict` instead? Or would that also break the tests? It doesn't seem like we need `OrderedDict` otherwise.

Reply:
We do not want to break API, the serializer returns OrderedDict now. It is why we use also OrderedDict